src/main/java/FileHandler.java

```diff
@@ -0,0 +1,106 @@
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+
+public class FileHandler {
+
+    public static ArrayList<String> Lines = new ArrayList<String>();
+
+    public static void initFiles(){
+        File dataDir = new File("./DataDir");
+        if (!dataDir.exists()) dataDir.mkdir();
```

Comment:
Consider enclosing the `dataDir.mkdir()` in braces as stated in the coding standard to maintain consistency and readibility.